web/templates/recipient_row.hbs

```diff
@@ -57,12 +57,27 @@
                 {{/if}}
             {{/if}}
 
-            {{#if stream_muted}}
-                <i class="zulip-icon zulip-icon-unmute stream_muted {{#if topic_unmuted}} on_hover_topic_mute {{else}} on_hover_topic_unmute {{/if}} recipient_bar_icon hidden-for-spectators" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" role="button" tabindex="0"
-                  {{#if topic_unmuted}} data-tooltip-template-id="topic-mute-tooltip-template" aria-label="{{t 'Mute topic' }}" {{else}} data-tooltip-template-id="topic-unmute-tooltip-template" aria-label="{{t 'Unmute topic' }}" {{/if}}></i>
+            {{#if development}}
+                <span class="change_visibility_policy hidden-for-spectators" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}">
+                    {{#if (eq visibility_policy all_visibility_policies.FOLLOWED)}}
+                        <i class="zulip-icon zulip-icon-follow recipient_bar_icon" data-tippy-content="{{t 'Mute or unmute topic'}}"
+                          role="button" tabindex="0" aria-haspopup="true" aria-label="{{t 'Mute or unmute topic' }}"></i>
+                    {{else if (eq visibility_policy all_visibility_policies.UNMUTED)}}
+                        <i class="zulip-icon zulip-icon-unmute-new recipient_bar_icon" data-tippy-content="{{t 'Mute or follow topic'}}"
+                          role="button" tabindex="0" aria-haspopup="true" aria-label="{{t 'Mute or follow topic' }}"></i>
+                    {{else}}
+                        <i class="zulip-icon zulip-icon-mute-new recipient_bar_icon" data-tippy-content="{{t 'Change visibility policy'}}"
+                          role="button" tabindex="0" aria-haspopup="true" aria-label="{{t 'Change visibility policy' }}"></i>
```

Comment:
A few notes. One is that this icon seems to be missing `cursor: pointer`. Next is that I think we want the icon to always show the current state -- for a topic in the default state, I think we want to use a "normal" icon, not a "mute" icon -- the latter should be reserved for if the topic is actually muted! 

And then finally, "visibility policy" is not a term intended to be user-facing. I think we want the tooltip to be something like a slightly fancier 2-line thing:

> You will get default notifications for this topic. (Or maybe in this case we just don't have a first line?)
> You have muted this topic.
> You have unmuted this topic.
> You follow this topic.

With a second like that's something like "Change notifications for this topic" as a click action description. I'm pretty sure we have an existing tooltip with this shape but cannot remember it. @amanagr do you recall?

Reply:
We use one in `message_inline_image_tooltip`.

Reply:
* icon seems to be missing `cursor: pointer` -- Done
* I think we want the icon to always show the current state
  > Earlier we had planned to NOT show 'default' in the popover (which was ofcourse confusing). I have updated the behavior as per [our discussion in CZO](https://chat.zulip.org/#narrow/stream/101-design/topic/Follow.20Topics.20UI/near/1624043). 
  Also, we are still [discussing the icons](https://chat.zulip.org/#narrow/stream/101-design/topic/UI.20redesign.3A.20recipient.20bar/near/1617263) to use. For now, I have kept the icons for "Default" and "Unmute" as the same. The changes are not visible outside the dev env; so we can update the icons (once finalized) in a small commit. 
* I think we want the tooltip to be something like a slightly fancier 2-line thing:
  
  <details><summary>Default</summary>
  <img src="https://github.com/zulip/zulip/assets/56781761/63e7e511-aa24-4874-83ff-29520584e8ef">
  </img>
  </details> 
  
  <details><summary>Muted</summary>
  <img src="https://github.com/zulip/zulip/assets/56781761/f9bdcda9-336f-486c-b5a7-c353e76cfaee">
  </img>
  </details> 
  
  <details><summary>Unmuted</summary>
  <img src="https://github.com/zulip/zulip/assets/56781761/6ab43a8e-7a13-496f-80a0-650d1607633f">
  </img>
  </details> 
  
  <details><summary>Followed</summary>
  <img src="https://github.com/zulip/zulip/assets/56781761/d559a6e2-d8d4-4a36-a457-927de4c45a70">
  </img>
  </details>

Reply:
Should we use "Click to change the state of this topic", because these visibility policies affect things other (visually) than notifications. 

We use the "state" term in 'SETTINGS/TOPICS' panel.

Reply:
I don't think "state" feels right -- let's discuss this detail in #design; I think this is a workable prototype but we'll probably refine it a bit more as we go.